server/views/partials/activities/allocation-dashboard/prisonerWaitlistHtml.njk

```diff
@@ -0,0 +1,430 @@
+    {% endfor %}
+
+{% set withdrawnApplicationRows = [] %}
+    {% for application in withdrawnApplications %}
+            {% set activityNameHtml%}
+                <ul class='govuk-list'>
+                    <li><a href='/activities/allocation-dashboard/{{ application.activityId }}' class='govuk-link govuk-link--no-visited-state' target='_blank'>{{ application.activity.activityName }}</a></li>
+                </ul>
+            {% endset %}
+            {% set requestDateHtml%}
+                <ul class='govuk-list'>
+                    <li>{{ application.requestedDate | formatDate('d MMMM yyyy') }}</li>
+                </ul>
+            {% endset %}
+            {% set requesterHtml%}
+                <ul class='govuk-list'>
+                    <li>{{ application.requestedBy | toTitleCase }} </li>
+                </ul>
+            {% endset %}
+            {% set viewWaitlistHtml %}
+                <ul class='govuk-list govuk-!-text-align-right'>
+                    <li><a href='/activities/allocation-dashboard/{{ application.activityId }}#waitlist-tab' class='govuk-link govuk-link--no-visited-state' target='_blank'>View waitlist</a></li>
+                </ul>
+            {% endset %}
+            {% set viewEditHtml %}
+                <ul class='govuk-list govuk-!-text-align-right'>
+                     <li><a href='/activities/waitlist/view-and-edit/{{application.id}}/view' class='govuk-link govuk-link--no-visited-state' target='_blank'>View or Edit application</a></li>
+                </ul>
+            {% endset %}
+
+            {% set withdrawnApplicationRows = (withdrawnApplicationRows.push([
+                    {
+                        html: activityNameHtml,
+                        attributes: {
+                            "data-sort-value": application.activity.activityName
+                        }
+                    },
+                    {
+                        html: requestDateHtml,
+                        attributes: {
+                            "data-sort-value": application.requestedDate | getUnixTime
+                        }
+                    },
+                    {
+                        html: requesterHtml,
+                        attributes: {
+                            "data-sort-value": application.requestedBy
+                        }
+                    },
+                    {
+                        html: viewWaitlistHtml
+                    },
+                    {
+                        html: viewEditHtml
+                    }               
+            ]), withdrawnApplicationRows) %}
+    {% endfor %}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full govuk-!-margin-bottom-7">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-full govuk-!-margin-bottom-7">
+                <h2 class="govuk-heading-m">Waitlist and applications</h2>
+            </div>
+        </div>
```

Comment:
When I run this locally, it feels like the margin on the bottom of the headings is too big. I think where you've used `govuk-!-margin-bottom-7` it could probably be a smaller margin? We probably don't need two of them nested together either.
Also it might be worth you having a look at where you've used `govuk-grid-row` as I don't think you need it twice here - you should just need one govuk-grid-row and then a column width class inside it. I think this should work...

`<div class="govuk-grid-row">
    <div class="govuk-grid-column-full govuk-!-margin-bottom-5">
            <div class="govuk-grid-column-full">
                <h2 class="govuk-heading-m">Waitlist and applications</h2>
            </div>
       ....
        </div>`